command/agent/agent.go

```diff
@@ -49,6 +56,8 @@ func NewAgent(config *Config, logOutput io.Writer) (*Agent, error) {
 		shutdownCh: make(chan struct{}),
 	}
 
+	a.createConsulConfig()
```

Comment:
Comment this. Kinda odd placement so it would be good to explain it initializes the config for both server and client